Example_App/www/js/index.js

```diff
@@ -155,18 +155,18 @@ function pushRegister() {
 }
 document
   .getElementById("in-app-consent-true")
-  .addEventListener("click", inAppUpdateConsentTrue);
+  .addEventListener("click", updateConsentForUserTrue);
```

Comment:
is this renaming good? 
in optimove sdk we have `OptimoveInApp.getInstance().updateConsentForUser(consented);`, but there it's clear that it's about inApp from `OptimoveInApp` 'context'

`updateConsentForUser` doesnt tell what this consent is about